src/main/webapp/js/datepicker.js

```diff
@@ -89,7 +93,8 @@ function getMaxDateForVisibleDate(startDate, publishDate) {
 
 /**
  * @assumption: visibleDate has a valid value
- * @returns {Date} visibleDate
+ * @param {Date} visibleDate
+ * @returns {Date}
  */
 function getMinDateForPublishDate(visibleDate) {
     return visibleDate;
```

Comment:
Seems the backend enforces the constraint while the font-end doesn't. If you want to fix, go ahead.